File: JavaScript/Lv_0/배열 비교하기.js
function solution(arr1, arr2) {
    if(arr1.length == arr2.length)
    {
        const a1 = arr1.reduce((a, c)=> a+=c);
        const a2 = arr2.reduce((a, c)=> a+=c);
        
        return a1 == a2 ? 0 : a1 > a2 ? 1 : -1 ;
    }
    else return arr1.length > arr2.length ? 1 : -1;
}

File: C#/Lv_0/배열 비교하기.cs
using System;
using System.Linq;

public class Solution {
    public int solution(int[] arr1, int[] arr2) {
        if(arr1.Length == arr2.Length)
        {
            int a1 = arr1.Sum();
            int a2 = arr2.Sum();
            return a1>a2 ? 1 : a1==a2 ? 0 : -1;
        }
        return arr1.Length > arr2.Length ? 1 : -1;
    }
}